<launch>
   <node pkg="cv_camera" type="cv_camera_node" name="cv_camera">
      <param name="device_id" value="1" />
      <param name="rate" value="10" />
   </node>

   <include file="$(find kinect2_bridge)/launch/kinect2_bridge.launch"/>
   
   <!--node name="kinect2_bridge" pkg="kinect2_bridge" type="kinect2_bridge" output="screen">
       <param name="publish_tf"        type="bool"   value="false"/>
       <param name="fps_limit"         type="double" value="10"/>
   </node-->
   
   <node name="blob_tracker" pkg="open_manipulator_visual_servoing" type="blob_tracking"/>
   <node name="cloud_sub" pkg="open_manipulator_visual_servoing" type="cloud_sub"/>
   <node name="color_finder" pkg="open_manipulator_visual_servoing" type="color_finder"/>
   <node name="object_checker" pkg="open_manipulator_visual_servoing" type="object_checker"/>
</launch>
